File: app/src/main/java/com/lazylee/lzywanandroid/mvp/BaseView.java
package com.lazylee.lzywanandroid.mvp;

/**
 * base view to impl
 * Created by lazylee on 2018/3/16.
 */

public interface BaseView<T extends BasePresenter> {

    void setPresenter(T presenter);

    void showMessage(String msg,int type);

    void showMessage(String msg);


}

File: app/src/main/java/com/lazylee/lzywanandroid/mvp/BaseView.kt
package com.lazylee.lzywanandroid.mvp

/**
 * base view to impl
 * Created by lazylee on 2018/3/16.
 */

interface BaseView<T : BasePresenter> {

    fun setPresenter(presenter: T)

    fun showMessage(msg: String, type: Int)

    fun showMessage(msg: String)


}
